tests/test_update_ssh_keys.rs

```diff
@@ -0,0 +1,499 @@
+// Copyright 2017-2023 Flatcar Authors
+
+#[cfg(test)]
+mod tests {
+    extern crate subprocess;
```

Comment:
Extern crate shouldn't be needed, or? Normally use is enough: https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html

Reply:
I had thought also that extern crate would not be needed.
But I had to add it, due to a strange build error that suggested adding that.

Now I figured out why.
This repo is soooo ancient, ~7 years old, that its `Cargo.toml` does not have an `edition`.
Adding `edition = "2021"` fixed such a strange issue.
See also https://github.com/rust-lang/vscode-rust/issues/513#issuecomment-472393831.

Reply:
Right, opting into a newer edition makes sense among other renovation.